stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/WebSocketUtil.java

```diff
@@ -180,7 +178,7 @@ public static void handleWebSocketCallback(Context context, CallableUnitCallback
      * @param uri URI to refactor.
      * @return refactored URI.
      */
-    public static String refactorUri(String uri) {
+    static String refactorUri(String uri) {
```

Comment:
Why are we removing public here? Even though it is used in the package for now they are better to remain public since then it is easier to find them in other packages if they are to be reused.